delft/applications/grobidTagger.py

```diff
@@ -128,15 +126,18 @@ def configure(model, architecture, output_path=None, max_sequence_length=-1, bat
         max_sequence_length = 3000
 
     if max_epoch == -1:
-        max_epoch = 100
+        max_epoch = 60
 
-    return batch_size, max_sequence_length, model_name, embeddings_name, max_epoch, multiprocessing, early_stop
+    if patience == -1:
+        patience = 5
+
+    return batch_size, max_sequence_length, model_name, embeddings_name, max_epoch, multiprocessing, early_stop, patience
 
 
 # train a GROBID model with all available data
 def train(model, embeddings_name=None, architecture=None, transformer=None, input_path=None, 
         output_path=None, features_indices=None, max_sequence_length=-1, batch_size=-1, max_epoch=-1, 
-        use_ELMo=False, incremental=False, input_model_path=None):
+        use_ELMo=False, incremental=False, input_model_path=None, patience=5):
```

Comment:
I think it should be here:
```
patience=-1
```
otherwise the command line parameter is never considered.